### PR TITLE
Add SSM Distributor

### PR DESCRIPTION
**Pull request by ryanjpayne**
Add stacksets to deploy SSM Associations in each region/account to automatically install the Falcon sensor on Windows and Linux instances.

### CODE REVIEW

**Comment by ryanjpayne** [comment]
@kkvinjam

File: templates/crowdstrike_init_stack.yaml [hunk 1]
```diff
@@ -460,7 +599,7 @@ Resources:
           - id: CKV_AWS_18
             comment: S3 access logs intentionally not enabled
     Properties:
-      BucketName: !Sub ${SourceS3BucketNamePrefix}-${AWS::AccountId}-${AWS::Region}
+      BucketName: !Sub ${SourceS3BucketNamePrefix}-${AWS::StackName}
```
Comment (kkvinjam) [inline templates/crowdstrike_init_stack.yaml:602]: Using `AWS::StackName` results in S3 bucket creation to fail, when there are captial letters in the stackname. This is not scalable, please reset to `${AWS::AccountId}-${AWS::Region}`

**Review by kkvinjam — CHANGES_REQUESTED** [review]

**Review by kkvinjam — APPROVED** [review]

**Review by kkvinjam — CHANGES_REQUESTED** [review]
There are few additional issues reported by the scanner. Chimed you the details, please review and fix them. Thanks

File: templates/aws_cspm_cloudformation_v2.json [hunk 2]
```diff
@@ -209,19 +233,44 @@
     "CrowdStrikeCloudTrail": {
       "Type": "AWS::CloudTrail::Trail",
       "Condition": "ShouldCreateIOACloudTrailResource",
+      "Metadata": {
+        "cfn_nag": {
```
Comment (kkvinjam) [inline templates/aws_cspm_cloudformation_v2.json:237]: Incorrect suppression. Example below. For additional information, refer to [the doc](https://www.checkov.io/2.Basics/Suppressing%20and%20Skipping%20Policies.html).

```
  MyDB:
    Metadata:
      checkov:
        skip:
          - id: "CKV_AWS_157"
            comment: "Ensure that RDS instances have Multi-AZ enabled"
    Type: "AWS::RDS::DBInstance"
    Properties:
```

File: templates/aws_cspm_cloudformation_v2.json [hunk 3]
```diff
@@ -343,6 +412,28 @@
     "CrowdStrikeSensorManagementLambda": {
       "Type": "AWS::Lambda::Function",
       "Condition": "ShouldCreateSensorManagementResources",
+      "Metadata": {
+        "cfn_nag": {
+          "rules_to_suppress": [
+            {
+              "id": "CKV_AWS_116",
```
Comment (kkvinjam) [inline templates/aws_cspm_cloudformation_v2.json:419]: Same as above. Incorrect suppression. Please fix.

**Review by kkvinjam — CHANGES_REQUESTED** [review]
In addition to the commented changes. Additional review and rule_supress needed at File: /templates/crowdstrike_init_stack.yaml:854-860

**Review by kkvinjam — APPROVED** [review]

**Pull request merged**